bin/centreon-sanitize-images.php

```diff
@@ -0,0 +1,318 @@
+ * permission to link this program with independent modules to produce an executable,
+ * regardless of the license terms of these independent modules, and to copy and
+ * distribute the resulting executable under terms of Centreon choice, provided that
+ * Centreon also meet, for each linked independent module, the terms  and conditions
+ * of the license of that module. An independent module is a module which is not
+ * derived from this program. If you modify this program, you may extend this
+ * exception to your version of the program, but you are not obliged to do so. If you
+ * do not wish to do so, delete this exception statement from your version.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+require(__DIR__ . '/../vendor/autoload.php');
+use enshrined\svgSanitize\Sanitizer;
+use Symfony\Component\Finder\Finder;
+
+// Get Script options
+$options = getopt('h::', ['help::']);
+
+if (isset($options['help']) || isset($options['h'])) {
+    usage();
+    exit(0);
+}
+
+$files = listImages();
+
+if (empty($files['svgImages']) && empty($files['invalidImages'])) {
+    echo PHP_EOL . "Nothing to do, everything is fine." . PHP_EOL;
+    exit(0);
+}
+
+if (!empty($files['svgImages'])) {
+    echo "The following SVGs can be sanitized to prevent any injections:"  . PHP_EOL;
+    foreach ($files['svgImages'] as $svgImage) {
+        $pattern = str_replace('/', '\/', __DIR__ . '/../www/img/media/');
+        echo preg_replace('/' . $pattern . '/', '', $svgImage) . PHP_EOL;
+    }
+    echo PHP_EOL;
+}
+
+if (!empty($files['invalidImages'])) {
+    echo "The following images have an invalid MIME type or a mismatch between MIME type and " .
+    "file extension and can be replaced by a generic image:" . PHP_EOL;
+    foreach ($files['invalidImages'] as $invalidImg) {
+        $pattern = str_replace('/', '\/', __DIR__ . '/../www/img/media/');
+        echo preg_replace('/' . $pattern . '/', '', $invalidImg) . PHP_EOL;
+    }
+    echo PHP_EOL;
+}
+
+$proceed = askQuestion('Would you like to proceed to sanitize ? [y/N]: ');
+if (strtolower($proceed) === 'y') {
+    foreach ($files['svgImages'] as $svgImage) {
+        try {
+            sanitizeSvg($svgImage);
+        } catch (\Exception $ex) {
+            echo "ERROR - " . $ex->getMessage();
+            exit(1);
+        }
+    }
+
+    foreach ($files['invalidImages'] as $invalidImg) {
+        try {
+            convertCorruptedImageOrFail($invalidImg);
+        } catch (\Exception $ex) {
+            echo "ERROR - " . $ex->getMessage();
+            exit(1);
+        }
+    }
+
+    echo "Sanitize done." . PHP_EOL;
+} else {
+    exit(0);
+}
+
+###############################
+#        COMMON FUNCTION      #
+###############################
+
+/**
+ * Undocumented function
+ *
+ * @param string $question
+ * @param boolean $hidden
+ * @return string
+ */
+function askQuestion(string $question, $hidden = false): string
```

Comment:
I think centreon-bot will not cry if functions are declared as anonymous :)
cause it avoids side effects by overriding existing functions
```suggestion
$askQuestion = function (string $question, $hidden = false): string
```